docs/components/modeler/using-web-and-desktop-modeler-together.md

```diff
@@ -0,0 +1,93 @@
+---
+id: using-web-and-desktop-modeler-together
+title: Using Web and Desktop Modeler together
+description: "Understand the implications of using Web Modeler and Desktop Modeler for modeling process diagrams."
+---
+
+[Web Modeler](/components/modeler/web-modeler/launch-web-modeler.md) and [Desktop Modeler](/components/modeler/desktop-modeler/index.md) are both tools for designing [BPMN](/components/modeler/bpmn/bpmn.md) diagrams, but they serve different purposes and shine in different scenarios.
+
+Web Modeler is great for collaborative, cloud-based process modeling. It allows teams to work together in real-time, manage versions, and store models centrally. It's especially useful when you're working in distributed teams or want tight integration with the Camunda SaaS platform.
```

Comment:
Reposting @yT0n1 comment, as it was linked to a commit and not the PR:

> Good. I am not sure about the saas vs. self-managed discussion. It's more a remote/hosted Engine cluster, vs C8 Run, no?

Reply:
I have updated the content of this sections @yT0n1. Let me know if it is more accurate now

Reply:
@bastiankoerber @wollefitz what do you think about the introduction of this page?